### PR TITLE
refactoring Koans and Art Gallery

### PR DESCRIPTION
**Pull request by lynnsamuelson**
Koans was changed to direct to the new repository and make the instructions match that project.  

For Art Gallery, notes were added for working with each team member having a local database and some requirements were moved to be bonus criteria

### CODE REVIEW

File: 1-foundations/exercises/FND_KOANS.md [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
-Koans are a series of puzzels designed to teach the contructs of the language.  Some people understand the structure of the koans faster than others. If you don't understand ask for help.  The main point is that you learn the basic rules of the language and not just go through and get the answers correct.  These will be a reference as you continue to learn C#.
+This exercise walks you through major concepts in C# and dotnet while introducing you to basic testing by  having you complete small steps to make a test pass.  Nobody likes this exersize, but it is an important step in leaning a new language.  
+
+Koans are a series of puzzels designed to teach the contructs of the language.  Some people understand the structure of the koans faster than others. If you don't understand ask for help.  The main point is that you learn the basic rules of the language and not just go through and get the tests to pass.  These will be a reference as you continue to learn C#.
+
 
 ###Getting Started
-* Fork the github project from: https://github.com/CoryFoy/DotNetKoans
+* Fork the github project from: https://github.com/lynnsamuelson/CSharpDotNetKoans
```
Comment (chortlehoort) [inline 1-foundations/exercises/FND_KOANS.md]: @lynnsamuelson I forked your repo into the NSS org. Can you replace the URL here with

https://github.com/nashville-software-school/CSharpDotNetKoans

File: 3-database-driven-applications/exercises/mvc/ArtGallery/MVC_ArtGallery.md [hunk 2]
```diff
@@ -7,25 +7,32 @@ As a customer I will:
...
 As product owners, we have provided:
 
-1. A partial ERD diagram 
+1. A partial ERD diagram - This is to use as a reference, but you will need to make changes to complete the project.
 2. Screen shots
-3. Spreadsheet 
+3. Spreadsheet of the data the gallery owner currently has that needs to be included.
+
+Notes: 
+ 1. Working on a team and having a local database can be challenging.  Be sure to have a stategy for this probelm early in the project.  One strategy is to script an sql file that will create the local database and add that data.  Remember to make sure that the tables are created in an order that does not add a foreign key before the table being referenced is created.  The same goes for the data.
```
Comment (chortlehoort) [inline 3-database-driven-applications/exercises/mvc/ArtGallery/MVC_ArtGallery.md]: My hope is that, in future cohorts, the existence of .NET Core will allow us to teach them about Code First development, and then all they will have to do it migrations. Perhaps add a hint to this strategy in here as well.

**Comment by lynnsamuelson** [comment]
@chortlehoort I made the changes, but am wondering how the forked koans will work.  Caitlin is working on a branch from my repo.  Will the changes be committed in the forked version too?

**Comment by chortlehoort** [comment]
I can sync from your repo (upstream) down to the NSS version once Caitlin's branch is complete.

**Comment by chortlehoort** [comment]
👍

**Pull request merged**